### PR TITLE
EmailAuthenticityCheck description update.

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/19557

## Description
Fixed an error in some arguments' description.

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/edit/EmailAuthenticityCheck/Scripts/CheckEmailAuthenticity/CheckEmailAuthenticity.yml?pr=%2Fdemisto%2Fcontent%2Fpull%2F4589)
- [ ] [CHANGELOG](https://github.com/demisto/content/edit/EmailAuthenticityCheck/Scripts/CheckEmailAuthenticity/CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F4589)

### CODE REVIEW

File: Scripts/CheckEmailAuthenticity/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
```
Comment (yaakovi) [inline Scripts/CheckEmailAuthenticity/CHANGELOG.md]: @kirbles19 those are the changes made here:
```
  - Updated descriptions of command arguments and outputs.
  - output paths will always be set
```

**Comment by orlichter1** [comment]
@yaakovi already did :)

File: Scripts/CheckEmailAuthenticity/CHANGELOG.md (hunk 1 above)
Comment (kirbles19) [reply Scripts/CheckEmailAuthenticity/CHANGELOG.md]: Output paths are always set (populated).

Not sure what "set" means...

Comment (yaakovi) [reply Scripts/CheckEmailAuthenticity/CHANGELOG.md]: @kirbles19 "populated" is good

Comment (yaakovi) [reply Scripts/CheckEmailAuthenticity/CHANGELOG.md]: @orlichter1 please fix the CHANGELOG accordingly

**Pull request merged**